lib/bemhtml/runtime/index.js

```diff
@@ -389,6 +392,25 @@ BEMHTML.prototype.runOne = function runOne(json) {
   return res;
 };
 
+BEMHTML.prototype.tryRun = function tryRun(context, ent) {
+  var res;
+
+  try {
+    res = ent.run(context);
+  } catch (e) {
+    console.error('BEMXJST ERROR: cannot render ' +
```

Comment:
@tadatuta context contains circular structure and many information useless for debugging in this case.